### DISCUSSION TITLE
I have a question.

### DISCUSSION THREAD

**Discussion by lsh3910** [cloudforet-io/community#127] [Q&A]
1. Even though I registered a GCP account, the message that the product ID is different continues to appear and is recorded as an error. what?
- Register the key by entering the product_id of the account correctly and extracting json.
- 
![image](https://github.com/cloudforet-io/cloudforet/assets/60865605/d1733254-3b3e-4468-b1af-5cedc96cf4f7)

2. Why do I not see the current status of the server/database on the dashboard screen after loading the Azure account information? And the cost part seems to be solved with AWS. I'd like to know how it happened.
- 
![image](https://github.com/cloudforet-io/cloudforet/assets/60865605/e8fddff9-d30b-4e55-8765-e9b9b827a538)

3. Please provide a guide document on how to install ingress from nginx so that it can be easily opened with ingress even in an on-premise environment.

**Comment by choonho** [comment]
For GCP error, there may be two possible errors.

One is miss configuration of GCP credentials. GCP credential.
I highly recommend to update GCP credentials by JSON, instead of one-by-one update.

![image](https://github.com/cloudforet-io/cloudforet/assets/3993088/f992942e-78ea-4c94-ae24-f10441f8ed37)

Second is permission of IAM.
I think that this is not case.

In the "Service Account"  of "Asset Inventory", you can update your gcp credential again.

**Reply by lsh3910** [reply]
Why don't you write down the answer to number 3?
And the answer to number 1 seems to repeat the same words too much. This is an error that appears even though the key in Json format was extracted and inserted.
Even in the introduction, it was said that the key was registered by extracting it as Json, but what is wrong?
You need to be specific about what you need from the credentials and what permissions you need.
The second one said that it was an IAM permission problem, but the corresponding part also needs what kind of permission is the problem.

that is what i need
We need specific answers about granting authority to resolve the contents in #1 and #2.
